pipelines/skylab/build_indices/modify_gtf_human.sh

```diff
@@ -0,0 +1,120 @@
+# Genome metadata
```

Comment:
@khajoue2 Might want to cite 10x for this code and point to their site.

Reply:
Added it to the script in the docker. This script should not be here.